allennlp/data/dataset_readers/snli.py

```diff
@@ -29,14 +29,29 @@ class SnliReader(DatasetReader):
         We use this `Tokenizer` for both the premise and the hypothesis.  See :class:`Tokenizer`.
     token_indexers : `Dict[str, TokenIndexer]`, optional (default=`{"tokens": SingleIdTokenIndexer()}`)
         We similarly use this for both the premise and the hypothesis.  See :class:`TokenIndexer`.
+    tokenize_separately : `bool`, optional
```

Comment:
I like having the default be a best guess like this, though the name is a bit confusing to me.  Maybe `combine_fields` or `use_single_field`, or something, with the default being `False`?

Reply:
The issue is this isn't necessarily about tokenization, it's more about how the data is represented.

Reply:
Do you mean the default being `None`? Because if it's `False` we can no longer take a guess.

Reply:
Right, sorry, I guess I just meant negating the logic in the guess.